.github/workflows/nightly.yaml

```diff
@@ -35,6 +35,21 @@ jobs:
           packager_tag: el8-go1.20
           version: nightly
 
+  el9_64:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          path: src/github.com/choria-io/aaasvc
```

Comment:
lets change this one - and the earlier one by el8 - to `src/github.com/choria-io/stream-replicator` otherwise LGTM

Reply:
Updated now - Also did el7. Can revert that but assumed you wanted all 3 changed